pkg/operator/resources/resources.go

```diff
@@ -72,26 +73,31 @@ func Deploy(projectBytes []byte, configPath string, configBytes []byte, force bo
 		ProjectByteMap: projectFileMap,
 		ConfigFilePath: configPath,
 	}
-	apiConfigs, err := spec.ExtractAPIConfigs(configBytes, types.AWSProviderType, projectFiles, configPath)
+
+	isProjectUploaded, err := config.AWS.IsS3File(config.Cluster.Bucket, projectKey)
```

Comment:
Why has this been reordered to write the project files to S3 before extracting api configs and validating?

Reply:
I think I moved it at an early stage...